src/domain/home/form/HomePreliminaryForm.tsx

```diff
@@ -83,7 +83,8 @@ const HomePreliminaryForm: FunctionComponent<Props> = ({
         initialValues={convertFormValues(stateFormValues)}
         onSubmit={handleSubmit}
         validate={validate}
-        isInitialValid={false}
+        initialErrors={{ childBirthdate: validateBirthdate('') }}
+        // To make sure form will not be valid to submit at first load
```

Comment:
Isn't it more normal to have the comment above the line it is about?

Reply:
Yea i will put it above